substrate/frame/support/procedural/src/construct_runtime/expand/origin.rs

```diff
@@ -209,6 +209,7 @@ pub fn expand_outer_origin(
 			system(#system_path::Origin<#runtime>),
 			#caller_variants
 			#[allow(dead_code)]
+			#[codec(skip)]
 			Void(#scrate::__private::Void)
```

Comment:
For context: The void type can't be instantiated, it can't be decoded as well. So skipping the variant keeps the same behavior.